src/Messenger/GenerateThumbnailsHandler.php

```diff
@@ -16,15 +16,14 @@
 use Sonata\MediaBundle\Model\MediaManagerInterface;
 use Sonata\MediaBundle\Provider\Pool;
 use Sonata\MediaBundle\Thumbnail\GenerableThumbnailInterface;
+use Symfony\Component\Messenger\Attribute\AsMessageHandler;
 use Symfony\Component\Messenger\Exception\UnrecoverableMessageHandlingException;
-use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
 
 /**
  * @author Jordi Sala Morales <jordism91@gmail.com>
- *
- * @psalm-suppress DeprecatedInterface
  */
-final class GenerateThumbnailsHandler implements MessageHandlerInterface
```

Comment:
This is a BC break for non SF 7 users.

We should use something like
https://github.com/sonata-project/SonataAdminBundle/blob/4.x/src/ArgumentResolver/CompatibleValueResolverInterface.php

```
if (interface_exists(MessageHandlerInterface::class)) {
    /** @internal */
    interface CompatibleMessageHandlerInterface extends MessageHandlerInterface
    {
    }
} else {
    /** @internal */
    interface CompatibleMessageHandlerInterface
    {
    }
}
```
and use it here:
```
final class GenerateThumbnailsHandler implements CompatibleMessageHandlerInterface
```